TombEngine/Game/effects/effects.h

```diff
@@ -160,6 +160,27 @@ struct Particle
 	int fxObj;
 	int roomNumber;
 	unsigned char nodeNumber; // ParticleNodeOffsetIDs enum.
+
+	// TODO: Clean above mess.
```

Comment:
Is this comment still needed?

Reply:
Yes, the commented section is to replace the implementation above.